developer-docs-site/docs/nodes/full-node/update-fullnode-with-new-releases.md

```diff
@@ -24,23 +27,29 @@ When `devnet` is wiped and updated with newer versions, you will need to update
 
 6. Update the configuration file (e.g., `fullnode.yaml`) with the new waypoint (if you configure the waypoint directly there).
 
-7. Restart the fullnode.
+7. Restart your local public fullnode by running the same start (`run`) command as before:
+  ```bash
+  cargo run -p aptos-node --release -- -f ./fullnode.yaml
+  ```
 
 8. See the [Verify initial synchronization](./fullnode-source-code-or-docker.md#verify-initial-synchronization) section for checking if the fullnode is syncing again.
 
 ## If you run a fullnode via Docker
 
-1. Shutdown your fullnode
+1. Stop your local public fullnode by running the below command:
+    ```bash
+    docker-compose down --volumes
+    ```
 2. Delete the entire directory which holds your fullnode config and data directory.
-3. Rerun the instructions on [Approach #2: Using Docker](./fullnode-source-code-or-docker.md#Approach-#2:-Using-Docker)
+3. Re-install and configure those files as during setup.
+4. Restart your local public fullnode by running the same start (`run`) command as before:
+  ```bash
+  cargo run -p aptos-node --release -- -f ./fullnode.yaml
+  ```
 
 ## If you run a fullnode on GCP
 
-Aptos devnet releases can be of two types: 
-- One with a data wipe to start over the Aptos blockchain
-- Second type is only a software update without a data wipe
-
-### Upgrade with data wipe
+### Upgrade with data wipe (devnet only)
```

Comment:
Nice! Maybe add a line just below this to make it obvious, e.g.,
`Upgrading your node in devnet requires a data wipe, as the network is reset on each deployment. Other networks (e.g., testnet and mainnet) don't require this step and we recommend not wiping your data in these networks`.